### PR TITLE
Frontend <-> Backend integration.

### PR DESCRIPTION
**Pull request by taier1**

### CODE REVIEW

**Review by spolel — DISMISSED** [review]

File: frontend/bower.json.orig [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jobadvisor",
+  "main": "index.html",
+  "dependencies": {
+    "polymer": "Polymer/polymer#^1.4.0",
+<<<<<<< HEAD
+    "paper-card": "PolymerElements/paper-card#^1.1.4"
+=======
```
Comment (camillomalnati) [inline frontend/bower.json.orig:8]: conflict comment from git, remove them pls

File: bin/installSeleniumAndChromeWebDriver [hunk 2]
```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+var http = require('http');
+var fs = require('fs');
+var child = require('child_process');
+
+var dest = 'selenium-server-standalone-2.48.2.jar';
```
Comment (camillomalnati) [inline bin/installSeleniumAndChromeWebDriver:7]: can't we use the last selenium version?

File: test/Freelancer/Freelancer.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      freelancer.lastName = 'Bubu';
+      freelancer.address = 'sasgfkb';
+      freelancer.email = 'askhb@as.askug';
+      freelancer.phone_number = '19821';
+      freelancer.location = 'Lugano';
+      freelancer.profession = 'IT guy';
+      utils.errorIfNullUndefinedOrEmpty(freelancer, 'firstName', done );
+    });
+
+    it('should fail if lastName is empty, null, or undefined', function(done){
```
Comment (camillomalnati) [inline test/Freelancer/Freelancer.js:75]: maybe change the data, looking all similar

File: test/backend/search.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+describe('Backend search tests', function(){
+
+  describe('GET /search', function(){
+
+    before(seed);
+    after(utils.dropDb);
+
+    it('should list all freelancers profiles matching the search criteria: profession', function(done) {
+
+      freelancer_testing = {
```
Comment (camillomalnati) [inline test/backend/search.js:23]: fix indent

**Review by camillomalnati — APPROVED** [review]
see comment for small changes

**Review by spolel — APPROVED** [review]

**Pull request merged**